### ISSUE TITLE
API2:2023 wording

### ISSUE THREAD

**Issue by securitylevelup** [OWASP/API-Security#94]
Did a diff on the API2:2019 and API2:2023 and noticed a specific change:
_A public-facing API is vulnerable if it:_

I would argue that public-facing is not necessary and applies to all types of APIs and would therefore recommend removing 'public-facing'.

**Comment by PauloASilva** [comment]
Hi @securitylevelup,
Would you like to open a PR with the suggested change?

Cheers,
Paulo A. Silva

**Comment by securitylevelup** [comment]
Of course, I wasn't sure if PRs were allowed from outside collaborators. Regardless, here it is: https://github.com/OWASP/API-Security/pull/103

**Comment by PauloASilva** [comment]
Thanks!

We will leave the 2023RC related issues and PRs open until the community feedback period ends so that anyone can jump into the discussion and improve whatever needs further improvements.

Cheers,
Paulo A. Silva
